sklearn/linear_model/tests/test_ridge.py

```diff
@@ -692,6 +692,36 @@ def test_sparse_design_with_sample_weights():
                                       decimal=6)
 
 
+def test_ridgecv_alphas():
+    # Test that no error is raised when fitting RidgeCV
+    # with integer alpha values and also
+    # check that an error is raised for negative alphas
+
+    X = np.array([[-1.0, -1.0], [-1.0, 0], [-.8, -1.0],
+                  [1.0, 1.0], [1.0, 0.0]])
+    y = [1, 1, 1, -1, -1]
+
+    # Integers
+    alphas = (1, 10, 100)
+    ridge = RidgeCV(alphas)
+    ridge.fit(X, y)
+
+    # Negative integers
```

Comment:
I would separate the tests for negative alphas since that they should raise error.
You can make a test called `test_ridgecv_neg_alphas()` with a parametrize pytest for the integer and floating type.